ocw/lib/EC2.py

```diff
@@ -395,6 +395,7 @@ def cleanup_k8s_jobs(self) -> None:
                         ret = client.list_job_for_all_namespaces(watch=False)
                         for job in ret.items:
                             age = (now - job.status.start_time).days
+                            self.log_err(f"{age}")
```

Comment:
leftover ? if yes remove it , if not please give some readable line which explains what you printing

Reply:
done